iaso/models/payments.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.translation import gettext_lazy as _
+
+
+class Statuses(models.TextChoices):
+    PENDING = "pending", _("Pending")
+    SENT = "sent", _("Sent")
+    REJECTED = "rejected", _("Rejected")
+
+
+class Payment(models.Model):
+    """
+    Model to store the status of payments linked to multiple OrgUnitChangeRequest by the same user.
+    """
+
+    status = models.CharField(choices=Statuses.choices, default=Statuses.PENDING, max_length=40)
+    change_requests = models.ManyToManyField("OrgUnitChangeRequest", related_name="payment")
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="payment")
```

Comment:
Is it the user who is gonna receive the payment ? Maybe we add a comment about that